### PR TITLE
[i ded] nerfs Derelict Outpost creature hatchlings

### PR DESCRIPTION
**Pull request by Y0SH1M4S73R**
## About The Pull Request

The Derelict Outpost ruin is full of mobs that at absolute most 3-shot most spacemen and have a very rapid AI. Most of this brought about by the map being untouched as additional abilities have been given to the type of mob that occupies it. This is far and away overtuned, so I made the "hatchling" mobs not have health-based damage and speed scaling.

As a bonus, you get map-level varedits replaced with typepaths.

## Why It's Good For The Game

Do space ruins need very-definitely-instant-death mobs, or are we good with just quick-death-if-you-cant-retreat-in-time mobs?

## Changelog

:cl:
balance: The hatchlings on the Derelict Outpost are marginally less deadly, to the extent that fighting them is slightly more feasible.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
These guys are weirdly nasty. I hadn't realized the health scaling component had been scaling their health inadvertently.

File: code/modules/mob/living/basic/space_fauna/netherworld/creature.dm [hunk 1]
```diff
@@ -100,3 +102,16 @@
...
 	enter_jaunt(cast_on)
+
+/mob/living/basic/creature/tiggles
+	name = "Miss Tiggles"
+
+/mob/living/basic/creature/hatchling
+	name = "hatchling"
+	health = 25
+	maxHealth = 25
+	health_scaling = FALSE
```
Comment (Ghommie) [inline code/modules/mob/living/basic/space_fauna/netherworld/creature.dm:113]:
```suggestion
	health_scaling = FALSE
	current_size = 0.85
```
I'm fixing an issue with initial current_size on #87021 by the way, if that's the reason why you're doing it on init.

File: code/modules/mob/living/basic/space_fauna/netherworld/creature.dm [hunk 2]
```diff
@@ -100,3 +102,16 @@
...
+
+/mob/living/basic/creature/hatchling/Initialize(mapload)
+	. = ..()
+	update_transform(0.85)
```
Comment (Ghommie) [inline code/modules/mob/living/basic/space_fauna/netherworld/creature.dm:117]:
```suggestion
```

**Comment by Livrah** [comment]
Is it one with M1911? Creatures guard just this part, I think it should be dangerous.

**Pull request merged**